src/web.ts

```diff
@@ -77,6 +77,3 @@ export class BranchDeepLinksWeb extends WebPlugin
 const BranchDeepLinks = new BranchDeepLinksWeb();
 
 export { BranchDeepLinks };
```

Comment:
```
const BranchDeepLinks = new BranchDeepLinksWeb();
export { BranchDeepLinks };
```
Let's remove these 2 lines, we no longer need to export an instance in capacitor 3.X
Instead, the new code at src/index.ts will handle plugin registration and exporting an instance.